extension/src/paymentHandler/payment-handler.js

```diff
@@ -12,7 +12,9 @@ async function handlePayment (paymentObject) {
     // if it's not adyen payment, ignore the payment
```

Comment:
not related to this branch, but could you remove lodash dependency from that file if we are not using it.
` error  '_' is assigned a value but never used`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/283/commits/ddde5580e2e422d302a20bc0195f3a9149b58a1f